python/mscclpp/__init__.py

```diff
@@ -6,6 +6,7 @@
 from ._mscclpp import (
     Communicator,
     Connection,
+    connect_nvls_collective,
```

Comment:
Do I miss something? Don't find where we use this function

Reply:
It's used in comm.py. Without it, compilation of allreduce_bench.py fails. 

I'm not sure if it's needed in both __init__.py and comm.py. Only comm.py seemed sufficient. But I wanted to allow more flexibility for future changes.

Reply:
I prefer to let it be as is, but if you insist, I can remove it from init.py. Please let me know.